views/heartbeat/two.hbs

```diff
@@ -10,9 +10,9 @@
 			<input type="button" name="people" id="peopleRefresh" value="People">
 			<input type="button" name="organisations" id="organisationsRefresh" value="Organisations">
 		</fieldset>
-		<script src="/static/js/d3.v5.js"></script>
-		<script src="/static/js/heartbeat.js"></script>
-		<script src="/static/js/heartbeatControls.js"></script>
+		<script src="/static/js/libs/d3.v5.js"></script>
```

Comment:
Could this block of scripts be a partial? Seems to be repeated in a few templates.